validators/node-ssl.adoc

```diff
@@ -0,0 +1,140 @@
+
+```shell
+skale ssl upload -c $PATH_TO_CERT_FILE -k $PATH_TO_KEY_FILE
+```
+
+[NOTE]
+Add -f flag to the command to override existing certificates on the node
+
+Once certificates are uploaded, you can check them by running:
+
+```shell
+skale ssl status
+```
+
+Full reference could be found here: https://github.com/skalenetwork/node-cli[skalenetwork/node-cli]. 
+
+=== Trusted authorities
+When choosing CA for getting your SSL certificate, consider a source that will be trusted by most major browsers, operating systems, and other devices.
+
+Here’s a list of links that can help you with your decision:
+
+* List of major CA providers: https://en.wikipedia.org/wiki/Certificate_authority#Providers
+* CAs trusted by Mozilla: https://wiki.mozilla.org/CA/Included_Certificates
+* CAs trusted by Apple: https://support.apple.com/en-us/HT204132, https://support.apple.com/en-us/HT209144
+
+Also, here’s a list of the most popular and well-known SSL providers:
+
+* DigiCert (https://www.digicert.com/)
+* GeoTrust (https://www.geotrust.com/)
+* RapidSSL (https://www.rapidssl.com/)
+
+Besides that, usually, you could purchase an SSL certificate from your domain provider. That could be the most convenient way for you to manage everything from a single place:
+
+* GoDaddy (https://godaddy.com/web-security/ssl-certificate)
+* NameCheap (http://namecheap.com/)
+* Name.com (https://www.name.com/ssl)
+
+Most of them usually providing you with a few options like RapidSSL and DigiCert.
+
+[WARNING]
+Certificates provided by Letsencrypt are widely used now and trusted by most operating systems and devices, so it’s OK to use them for your node, but don't forget about renewals since Letsencrypt certs expire in 3 months. You can read more about Letsencrypt security and trust here: https://letsencrypt.org/certificates/ and https://letsencrypt.org/2018/08/06/trusted-by-all-major-root-programs.html
+ 
+[NOTE]
+Generally, Letsencrypt SSL certs are considered safe, but there are some opponents to the idea of a 'free SSL certs for everybody': https://medium.com/swlh/why-lets-encrypt-is-a-really-really-really-bad-idea-d69308887801
+
+=== Reference Letsencrypt tutorial
+
+This brief tutorial shows you how to generate a wildcard SSL using Letsencrypt.
+
+Step 1. Install Certbot::
+https://certbot.eff.org/lets-encrypt/ubuntufocal-nginx
+
+Step 2. Run::
+
+```shell
+certbot certonly --standalone -d my.domain.com
+```
+
+Step 3. Copy .pem files to secure place:: 
+```shell
+cp *.pem ~/[SECURE_DIR]
+```
+
+=== Setting domain name of a registered node
+
+To set the domain name of the registered node with SKALE manager, execute the following node-cli command on each respective node:
+
+```shell
+skale node set-domain -d node-0.awesome-validator.com
+```
+
+=== Registering domain name for a new node
```

Comment:
can we change to `Registering a new node with a domain name `

Reply:
changed doc layout